template/memory-cards.js

```diff
@@ -1,21 +1,31 @@
-// See cardset-example.js for examples
-
 var MemoryCards = (function() {
-	function Ctor() {
-		//...
-		this.values = function() {
-
-		}
 
-		this.match = function(a,b) {
-
-		}
+	var dragons = [
+    ['shadow', 1], ['shadow', 2],
+    ['horned', 1], ['horned', 2],
+    ['black', 1], ['black', 2],
+    ['blue', 1], ['blue', 2],
+    ['green', 1], ['green', 2],
+    ['red', 1], ['red', 2],
+    ['white', 1], ['white', 2],
+    ['brass', 1], ['brass', 2],
+    ['bronze', 1], ['bronze', 2],
+    ['copper', 1], ['copper', 2],
+    ['gold', 1], ['gold', 2],
+    ['silver', 1], ['silver', 2]
+  ];
 
```

Comment:
When I first tested it out I thought that it was a color matching game and then I was really confused when I got to "horned" and "shadow." Now I see that they are dragons! Are you going to add any pictures or anything to boost the dragon-ness of it? 

Aside from that everything looks good. Eric has a few suggestions about functionality. Nice work!